src/Admin/components/PrivateNotes/index.tsx

```diff
@@ -1,16 +1,16 @@
-import {__} from '@wordpress/i18n';
-import {addQueryArgs} from '@wordpress/url';
-import useSWR from 'swr';
-import {useState} from 'react';
+import ModalDialog from '@givewp/components/AdminUI/ModalDialog';
+import Header from '@givewp/src/Admin/components/Header';
```

Comment:
```suggestion
import {Header} from '@givewp/admin/components';
```

Reply:
Resolved in c58a06a59ad3cc1102089f6883767bb7820eb7dc